docs/parametertable.rst

```diff
@@ -0,0 +1,570 @@
+.. list-table::
```

Comment:
Can this file be generated as part of `make docs`? Or at least can its regeneration appear in the Makefile?

Reply:
Currently, this file does not generate when `make docs`. I plan to have it after 0.8. Right now, I would like to keep a static version of this file.